ClusteringAlgs/HierarchyCluster/BlossomCluster.py

```diff
@@ -1,21 +1,57 @@
 from typing import List, Set, Tuple
+import math
 import numpy as np
 import networkx as nx
 
 from MatrixGenerators import ReducedMatrix
 
 
+def is_equal(l1, l2):
+    l1.sort(key=len)
+    l2.sort(key=len)
+    if l1 == l2:
+        return True
+    return False
+
+
 class BlossomCluster:
     @staticmethod
-    def __blossom_match(matrix: np.ndarray) -> List[Tuple[int, ...]]:
+    def __blossom_match(matrix: np.ndarray, matches_percent: float = 0.50) -> List[Tuple[int, ...]]:
```

Comment:
change naming convention to max_weight matching. refactor function.
lowest function for getting pure matching
function for choosing best matchings
aid function to organize return value (if needed).